group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/TargetAssignmentBuilderTest.java

```diff
@@ -145,13 +147,13 @@ public TargetAssignmentBuilder.TargetAssignmentResult build() {
             Map<String, AssignmentMemberSpec> memberSpecs = new HashMap<>();
 
             // All the existing members are prepared.
-            members.forEach((memberId, member) -> {
-                memberSpecs.put(memberId, createAssignmentMemberSpec(
+            members.forEach((memberId, member) -> memberSpecs.put(memberId,
+                createAssignmentMemberSpec(
                     member,
                     targetAssignment.getOrDefault(memberId, Assignment.EMPTY),
                     subscriptionMetadata
-                ));
-            });
+                )
```

Comment:
I think this is more readable since the next lines are arguments for createAssignmentMemberSpec and the way above I have to look a lot from right to left to see what the arguments are for. I feel like this makes more sense here